tests/helpers/test_llm.py

```diff
@@ -1267,3 +1276,26 @@ async def test_calendar_get_events_tool(hass: HomeAssistant) -> None:
         "start_date_time": now,
         "end_date_time": dt_util.start_of_local_day() + timedelta(days=7),
     }
+
+
+async def test_get_home_state_no_entities(hass: HomeAssistant) -> None:
```

Comment:
Should we even expose the tool if no entities are exposed?

Reply:
You're right, in the code its no tsupposed to be exposed wen there are no entities exposed. It's only added a block `if exposed_entities:` so maybe there actually is an entity exposed here or something. I'll double check.

Reply:
Fixed, the check was wrong.